CMakeLists.txt

```diff
@@ -1422,6 +1424,19 @@ else()
 endif()
 
 
+################################################################################
+# ION GNSS-SDR Metadata Standard
+################################################################################
+include(FetchContent)
+FetchContent_Declare(
+        gnss_metadata_standard
+        GIT_REPOSITORY https://github.com/IonMetadataWorkingGroup/GNSS-Metadata-Standard
+        GIT_TAG master
```

Comment:
Honestly, I wanted to use their library since it is their standard and you would think it would be in a better state. But the latest commit on their `devel` branch is "Fix Memory Leak", it's 4 years old and hasn't been merged yet.

I will fix a commit, since there are no tags apart from the branches themselves.

Reply:
Fixed: pointing to the current last commit in the `master` branch.